.gitignore

```diff
@@ -26,4 +26,5 @@ unit_testing/*log
 sandbox.py
 
 config/nemo_tst_gridded.json
-
+.coverage
```

Comment:
This will need some explaining somewhere. Perhaps a ticket to add some docs

Reply:
Docs already added to develop branch of site